curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -275,69 +275,97 @@ int FileCacheManager::Read(uint64_t inodeId, uint64_t offset, uint64_t length,
         return readOffset;
     }
 
-    std::vector<S3ReadRequest> totalS3Requests;
-    auto iter = totalRequests.begin();
-    uint64_t fileLen;
     {
+        unsigned int maxRetry = 3;  // hardcode, fixme
```

Comment:
Retry 3 times is not enough maybe

Reply:
we only retry when cache is stale, retry once(clear cache and get inode) and success is expected.
i think 3 is enough. will make it a config later, but later.